sky/packages/sky/lib/src/widgets/framework.dart

```diff
@@ -468,15 +502,6 @@ abstract class Element<T extends Widget> implements BuildContext {
   /// Calls the argument for each child. Must be overridden by subclasses that support having children.
   void visitChildren(ElementVisitor visitor) { }
 
-  /// Calls the argument for each descendant, depth-first pre-order.
```

Comment:
make sure none of the demos or tests use this... I'm pretty sure I've tried to use it in the past (though I may have fallen back on visitChildren).